### PR TITLE
Fix Thumb symbols bitness for Mach-O ##bin

### PR DESCRIPTION
**Pull request by GustavoLCR**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Test plan**

Tests pass

### CODE REVIEW

File: test/db/anal/mach0 [hunk 1]
```diff
@@ -7,37 +7,37 @@ pd 17 @0x8058
 EOF
 EXPECT=<<EOF
 |           0x00008058      .dword 0x0000e403
-            ; DATA XREF from sub.free_7418 @ 0x7d30
+            ; DATA XREF from sub.free_7890 @ 0x7d30
```
Comment (ret2libc) [inline test/db/anal/mach0:10]: do you know why this change?

**Comment by XVilka** [comment]
@mrmacete could you please take a look?

File: test/db/anal/mach0 (hunk 1 above)
Comment (trufae) [reply test/db/anal/mach0:10]: yes, the change is correct. well the old 7418 is wrong for sure.

Actually drawing the graph of this function spits out some warnings:

```
WARNING: r_agraph_add_edge: assertion 'g && a && b' failed (line 3823)
WARNING: r_agraph_add_edge: assertion 'g && a && b' failed (line 3823)
```

imho this PR is good to merge. maybe @mrmacete can give a 2nd opinion, but as it's a small step toward improving the arm analysis.

Comment (trufae) [reply test/db/anal/mach0:10]: i say that is wrong for sure because at 7418 there's no begining of function, it's in the middle of another function, this was caused by wrong thumb/arm code analysis.
